do-app-platform-op-cli/README.md

```diff
@@ -91,8 +91,7 @@ Copy the following command, replace `https://op-scim-bridge-example.ondigitaloce
 
 ```bash
 curl --silent --show-error --request GET --header "Accept: application/json" --header "Authorization: Bearer $(
-  op read op://${VAULT:-op-scim}/${ITEM:-"bearer token"}/credential)"
-) https://op-scim-bridge-example.ondigitalocean.app/health
+  op read op://${VAULT:-op-scim}/${ITEM:-"bearer token"}/credential)" https://op-scim-bridge-example.ondigitalocean.app/health
```

Comment:
Little nitpick:

```suggestion
  op read op://${VAULT:-op-scim}/${ITEM:-"bearer token"}/credential
)" https://op-scim-bridge-example.ondigitalocean.app/health
```

No change to functionality, but I prefer to keep the line break for readability. :-)